packages/web-components/src/index.ts

```diff
@@ -169,6 +169,12 @@ import {
   register as registerTooltip,
 } from "./tooltip/index.ts";
 
+import {
```

Comment:
I suggest removing the line between imports to let our prettier sort them.

```suggestion
} from "./tooltip/index.ts";
import {
```